recipe/meta.yaml

```diff
@@ -24,24 +24,21 @@ requirements:
     - python
     - cachecontrol >=0.12.4,<0.13.0
     # cachecontrol:filecache
-    - lockfile >=0.9
```

Comment:
I'm not sure this is true. poetry install `cachecontrol[filecache]` and that requires `lockfile >=0.9`. Conda does not have the concept of optional packages and this kind of specification upstream is hard to get it right.

Reply:
ah, that makes sense! I didn't take transitive optional packages into account.

I'll have another look tomorrow. Thanks